components/sysutils/coreutils/test/results-i386.master

```diff
@@ -1,11 +1,8 @@
-FAIL: tests/misc/dircolors.pl
-# TOTAL: 616
+# TOTAL: 621
 # XFAIL: 0
-# FAIL:  1
+# FAIL:  0
 # ERROR: 0
-FAIL: tests/misc/dircolors
-FAIL tests/misc/dircolors.pl (exit status: 1)
-# TOTAL: 616
+# TOTAL: 349
 # XFAIL: 0
-# FAIL:  1
+# FAIL:  0
 # ERROR: 0
```

Comment:
My gmake test results look quite different. I am not sure what the reason is for that:
+FAIL: tests/rm/cycle.sh
+FAIL: tests/rm/isatty.sh
+FAIL: tests/rm/fail-eacces.sh
+FAIL: tests/rm/rm1.sh
+FAIL: tests/rm/rm3.sh
+FAIL: tests/rm/unread2.sh
+FAIL: tests/rm/unreadable.pl
+FAIL: tests/misc/shred-remove.sh
+FAIL: tests/misc/sort-exit-early.sh
+FAIL: tests/cp/fail-perm.sh
+FAIL: tests/du/inacc-dir.sh
+FAIL: tests/mv/hard-3.sh
+FAIL: tests/mv/i-2.sh
+FAIL: tests/mv/i-3.sh
+FAIL: tests/mv/part-fail.sh
+FAIL: tests/mv/perm-1.sh
+ERROR: tests/rmdir/ignore.sh
 # TOTAL: 621
 # XFAIL: 0
-# FAIL:  0
-# ERROR: 0
-# TOTAL: 349
+# FAIL:  16
+# ERROR: 1
+FAIL: tests/rm/cycle
+FAIL tests/rm/cycle.sh (exit status: 1)
+FAIL: tests/rm/isatty
+FAIL tests/rm/isatty.sh (exit status: 1)
+FAIL: tests/rm/fail-eacces
+FAIL tests/rm/fail-eacces.sh (exit status: 1)
+FAIL: tests/rm/rm1
+FAIL tests/rm/rm1.sh (exit status: 1)
+FAIL: tests/rm/rm3
+FAIL tests/rm/rm3.sh (exit status: 1)
+FAIL: tests/rm/unread2
+FAIL tests/rm/unread2.sh (exit status: 1)
+FAIL: tests/rm/unreadable
+FAIL tests/rm/unreadable.pl (exit status: 1)
+FAIL: tests/misc/shred-remove
+FAIL tests/misc/shred-remove.sh (exit status: 1)
+FAIL: tests/misc/sort-exit-early
++ SORT_FAILURE=2
+FAIL tests/misc/sort-exit-early.sh (exit status: 1)
+FAIL: tests/cp/fail-perm
+FAIL tests/cp/fail-perm.sh (exit status: 1)
+FAIL: tests/du/inacc-dir
+FAIL tests/du/inacc-dir.sh (exit status: 1)
+FAIL: tests/mv/hard-3
+FAIL tests/mv/hard-3.sh (exit status: 1)
+FAIL: tests/mv/i-2
+FAIL tests/mv/i-2.sh (exit status: 1)
+FAIL: tests/mv/i-3
+FAIL tests/mv/i-3.sh (exit status: 1)
+FAIL: tests/mv/part-fail
+FAIL tests/mv/part-fail.sh (exit status: 1)
+FAIL: tests/mv/perm-1
+FAIL tests/mv/perm-1.sh (exit status: 1)
+ERROR: tests/rmdir/ignore
+ERROR tests/rmdir/ignore.sh (exit status: 99)
+# TOTAL: 621
 # XFAIL: 0
-# FAIL:  0
-# ERROR: 0
+# FAIL:  16
+# ERROR: 1